pyiron_base/jobs/job/core.py

```diff
@@ -707,10 +707,15 @@ def _internal_copy_to(
 
         # Copy files outside the HDF5 file
         if copy_files and os.path.exists(self.working_directory):
-            if len(os.listdir(new_job_core.working_directory)) == 0:
+            wd_content = os.listdir(new_job_core.working_directory)
+            if len(wd_content) == 0:
                 os.rmdir(new_job_core.working_directory)
+            elif wd_content == ["WARN_pyiron_modified_content"]:
```

Comment:
```suggestion
            elif wd_content == ["WARNING_pyiron_modified_content"]:
```

Just my preferred phrasing; if you hate it, then just leave it as-is, it's not bad.